bin/pdc_client

```diff
@@ -180,5 +182,26 @@ if __name__ == "__main__":
     else:
         with RequestMethod(client, options.resource,
                            options.traceback, options.debug) as request:
-            print(json.dumps(request.dispatch(options.request, data),
+            if options.all:
+                print("You choose get all the data!")
```

Comment:
I just want to give a comment about showing all the data.
But you are right, should just return data, I have removed it.